### PR TITLE
i18n(es): Update `authoring-content`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- What does this PR change? Give us a brief description.
Updated `authoring-content`

### CODE REVIEW

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia!

**Comment by dreyfus92** [comment]
Muchas gracias por la revisión @Waxer59 😁

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

File: docs/src/content/docs/es/guides/authoring-content.md [hunk 1]
```diff
@@ -201,9 +201,148 @@ var fun = function lang(l) {
...
+
+  ```js "Términos individuales" /Even.*supported/
+  // Términos individuales también pueden ser resaltados
+  function demo() {
+    return 'También las expresiones regulares son compatibles'
+  }
+  ```
+
+  ````md
+  ```js "Términos individuales" /Even.*supported/
```
Comment (delucis) [inline docs/src/content/docs/es/guides/authoring-content.md:251]: I think the regular expression here needs to be updated to match the translated code:

`````suggestion
  ```js "Términos individuales" /También.*compatibles/
  // Términos individuales también pueden ser resaltados
  function demo() {
    return 'También las expresiones regulares son compatibles'
  }
  ```

  ````md
  ```js "Términos individuales" /También.*compatibles/
`````

File: docs/src/content/docs/es/guides/authoring-content.md [hunk 2]
```diff
@@ -201,9 +201,148 @@ var fun = function lang(l) {
...
+  ```js "return true;" ins="inserted" del="deleted"
+  function demo() {
+    console.log('Estos son tipos de marcadores insertados y eliminados');
+    // La declaración de retorno utiliza el tipo de marcador predeterminado
+    return true;
+  }
+  ```
+
+  ````md
+  ```js "return true;" ins="inserted" del="deleted"
```
Comment (delucis) [inline docs/src/content/docs/es/guides/authoring-content.md:270]: Same here:

`````suggestion
  ```js "return true;" ins="insertados" del="eliminados"
  function demo() {
    console.log('Estos son tipos de marcadores insertados y eliminados');
    // La declaración de retorno utiliza el tipo de marcador predeterminado
    return true;
  }
  ```

  ````md
  ```js "return true;" ins="insertados" del="eliminados"
`````

**Review by delucis — COMMENTED** [review]
Thank you for the translation! Spotted two details to check.

**Comment by dreyfus92** [comment]
Thank you @delucis & @Waxer59 for your reviews 🫡

**Review by delucis — APPROVED** [review]
Let’s go!

**Pull request merged**